### ISSUE TITLE
is this repo maintained now?

### ISSUE THREAD

**Issue by athulantony95** [cusspvz/react-native-ipfs#5]
I see the golang and ipfs APIs are too old for this repo. Is this maintained now?

**Comment by cusspvz** [comment]
Nope. Ideally, I should make some time to adapt this to use the https://github.com/ipfs-shipyard/gomobile-ipfs core's interface.

**Comment by cusspvz** [comment]
If you're up to contribute we can refine the efforts together, plan and work on that.
